#!/usr/bin/env python
PACKAGE = "ros_cutball"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("red_H1",    int_t,    0, "An Integer parameter", 169, 0, 180)
gen.add("red_S1",    int_t,    0, "An Integer parameter", 210, 0, 255)
gen.add("red_V1",    int_t,    0, "An Integer parameter", 68, 0, 255)
gen.add("red_H2",    int_t,    0, "An Integer parameter", 180, 0, 180)
gen.add("red_S2",    int_t,    0, "An Integer parameter", 255, 0, 255)
gen.add("red_V2",    int_t,    0, "An Integer parameter", 255, 0, 255)


exit(gen.generate(PACKAGE, "ros_cutball", "red_hsv_"))